libs/core/src/lib/select/select.component.scss

```diff
@@ -93,3 +94,37 @@
         display: block;
     }
 }
+
+// no border class csss overrides the border of the button after adding important 
+.fd-button--noborder{
+   border: none  !important;
```

Comment:
can't we use light button instead?

Reply:
i tried using the light button but css doesn't overrides the look and feel, and also we have 3 other types of button for this component

Reply:
@Lokanathan-k  why we even changing core class ?
 is this is the reason? 

`in the core component there i have added option to add new select button type where user can add [selectType] and [gylph] assign diff types of select type and render the ui accordingly.
`

Can we just use triggerTemplate for this ? Where application can define their own format do we need to change the core at all? 

Also we should not be adding any enhacements that comes from our products. This is new direction. 

This PR is getting little bit messy... It woudl be cleaner to create separate PRs that updates A, and PR that changes B.

Reply:
we are not changing the core class but just adding another class with the property that  needs to be overridden.

when we write the class on the wrapper component it doesn't get overridden it add another layer of styling.